tutorials/install-psql-tutorial.md

```diff
@@ -0,0 +1,151 @@
+
+```bash
+sudo apt-get update
+sudo apt-get install postgresql-client
+```
+
+>:TIP: This only installs the `psql` client and not the PostgreSQL database.
+
+### Install on Windows 10
+We recommend using the installer from [PostgreSQL.org][windows-installer].
+
+### Last step: Connect to your PostgreSQL server
+Let’s confirm that `psql` is installed:
+
+```bash
+psql --version
+```
+
+Now, in order to connect to your PostgreSQL server, we’ll need the following 
+connection parameters:
+- Hostname
+- Port
+- Username
+- Password
+- Database name
+
+For example, if you're using [Timescale Cloud][timescale-cloud-install], you can 
+navigate to the ‘Overview Tab’ of your dashboard and locate your `host`, `port`, 
+and `password`, as highlighted below.
+
+<img class="main-content__illustration" src="https://s3.amazonaws.com/docs.timescale.com/hello-timescale/NYC_figure1_1.png" alt="NYC Taxis"/>
+
+There are two ways to use these parameters to connect to your PostgreSQL database.
+
+#### Option 1: Supply parameters at the command line
+In this method, you'd use parameter flags on the command line to supply the required
+information to connect to a PostgreSQL database: 
+
+```bash
+psql -h [HOSTNAME] -p [PORT] -U [USERNAME] -W -d [DATABASENAME]
+```
+
+Once you run that command, the prompt will ask you for your password. (This is the purpose 
+of the `-W` flag.)
+
+#### Option 2: Use a service URI
+The Service URI often begins with `postgres://`. You can see an example of a Service URI
+in the [Timescale Cloud][timescale-cloud-install] dashboard above.
+
+```bash
+psql postgres://[USERNAME]:[PASSWORD]@[HOSTNAME]:[PORT]/[DATABASENAME]?sslmode=require
+```
+
+### Fun things to do with psql
+
+#### Common psql commands
+Here is a table of common commands you'll find yourself using a lot:
+
+| Command       |      Actions                             |
+|---------------|------------------------------------------|
+|`\l`	        | List available databases                 |
+|`\c dbname`    | Connect to a new database                |
+|`\dt`	        | List available tables                    |
+|`\d tablename` | Describe the details of given table      |
+|`\dn`          | List all schemas in the current database |
+|`\df`          | List functions in the current database   |
+|`\h`           | Get a help on syntax of SQL commands     |
+|`\?`           | Lists all `psql` slash commands          |
+|`\set`         | System variables list                    |
+|`\timing`      | Shows how long a query took to execute   |
+|`\x`           | Show expanded query results              |
+|`\q`           | Quit `psql`                              |
+
+#### Save results of a query to a comma-separated file
+You may often find yourself running SQL queries with lengthy results. You can save these
+results to a comma-separated file (CSV) using the `COPY` command:
+
+```bash
+COPY (SELECT * FROM ...) TO '/tmp/myoutput.csv' (format CSV);
+```
```

Comment:
This is serverside copy most users will probably want clientside copy `\copy` though